Lib/tempfile.py

```diff
@@ -495,13 +501,31 @@ def __enter__(self):
         self.file.__enter__()
         return self
 
+    def _cleanup(self):
```

Comment:
I forgot to cache the `unlink()` function locally for the case when the `os` module is cleared at shutdown:

```suggestion
    def _cleanup(self, unlink=_os.unlink):
```